### PR TITLE
Create Duplication Reduction Charter

### PR DESCRIPTION
**Pull request by jared-lambert**
A first draft of a charter for the duplication reduction working group

Part of #171

### CODE REVIEW

File: docs/formation/governance/working-groups/duplication-reduction/duplication-reduction-charter.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Our goal is to ensure the CCC efforts do not duplicate pre-existing standards, and instead are net new contributions to the broader ecosystem aligned to the specific needs of the financial services industry.
+
+## Approach
+
+This work will span 2 main outputs:
+
+### 1 — Investigate existing frameworks
+
+The first step is to identify a set of existing frameworks and capabilities that we should investigate (eg. NIST, ISO, OSCAL, MITRE, etc) against the CCC goals. 
+
```
Comment (AdrianHammond) [inline docs/formation/governance/working-groups/duplication-reduction/duplication-reduction-charter.md:18]: Just had a quick look and couldn't find anything that documents the goals of CCC. The steering group charter has mission/objectives (https://github.com/finos/common-cloud-controls/blob/main/docs/governance/steering/charter.md) are the project goals documented anywhere?

**Review by AdrianHammond — COMMENTED** [review]
LGTM made one comment on CCC goals

File: docs/formation/governance/working-groups/duplication-reduction/duplication-reduction-charter.md (hunk 1 above)
Comment (eddie-knight) [reply docs/formation/governance/working-groups/duplication-reduction/duplication-reduction-charter.md:18]: Perhaps `CCC [WG] Goals` would suit better?

**Review by AdrianHammond — COMMENTED** [review]
LGTM +1

**Comment by rgriffiths-scottlogic** [comment]
+1

**Comment by jonmuk** [comment]
LGTM +1

**Comment by eddie-knight** [comment]
+1

**Comment by jared-lambert** [comment]
I've opened a ticket to figure out what's up with the CLA, I've signed it. -- https://jira.linuxfoundation.org/plugins/servlet/desk/portal/4/SUPPORT-27546

**Review by eddie-knight — APPROVED** [review]

**Comment by eddie-knight** [comment]
@robmoffat there is a bug in the CODEOWNERS preventing this from merge. Can you override and push it through?

**Comment by sshiells-scottlogic** [comment]
@jared-lambert  @eddie-knight  @robmoffat  what do we need to do to get this PR through?

**Comment by eddie-knight** [comment]
A placeholder file was added while this PR is pending. One of us will need to update the branch and resolve the conflict.

**Pull request merged**